build/custom.mk

```diff
@@ -3,14 +3,13 @@
 # If there's no MM_RUDDER_PLUGINS_PROD, add DEV data
 RUDDER_WRITE_KEY = 1d5bMvdrfWClLxgK1FvV3s4U1tg
 ifdef MM_RUDDER_PLUGINS_PROD
-RUDDER_WRITE_KEY = $(MM_RUDDER_PLUGINS_PROD)
+	RUDDER_WRITE_KEY = $(MM_RUDDER_PLUGINS_PROD)
 endif
 
-GO_BUILD_FLAGS += -ldflags '-X "github.com/mattermost/mattermost-plugin-api/experimental/telemetry.rudderWriteKey=$(RUDDER_WRITE_KEY)"'
-
+LDFLAGS += -X "github.com/mattermost/mattermost-plugin-api/experimental/telemetry.rudderWriteKey=$(RUDDER_WRITE_KEY)"
 ifdef PLUGIN_E2E_MOCK_OAUTH_SERVER_URL
-	GO_BUILD_FLAGS += -ldflags '-X "github.com/mattermost/mattermost-plugin-github/server/plugin.testOAuthServerURL=$(PLUGIN_E2E_MOCK_OAUTH_SERVER_URL)"'
+	LDFLAGS +=  -X "github.com/mattermost/mattermost-plugin-github/server/plugin.testOAuthServerURL=$(PLUGIN_E2E_MOCK_OAUTH_SERVER_URL)"
 endif
-.PHONY: deploy-e2e
-deploy-e2e: dist
-	E2E_TESTING=true PLUGIN_E2E_MOCK_OAUTH_SERVER_URL=http://localhost:8080 ./build/bin/pluginctl deploy $(PLUGIN_ID) dist/$(BUNDLE_NAME)
```

Comment:
removed this since it's not working, vars are needed at dist not at deploy command

duplicating dist info dist and sit-e2e seemed a bit risky to me